docs/cri/config.md

```diff
@@ -4,23 +4,14 @@ The CRI plugin config is part of the containerd config (default
 path: `/etc/containerd/config.toml`).
 
 See [here](https://github.com/containerd/containerd/blob/main/docs/ops.md)
-for more information about containerd config.
+for more information for Ops and Admins
+
+See [here](https://github.com/containerd/containerd/blob/main/docs/man/containerd-config.toml.5.md)
+for more information about the Containerd Configurations Guide
 
 Note that the `[plugins."io.containerd.grpc.v1.cri"]` section is specific to CRI,
 and not recognized by other containerd clients such as `ctr`, `nerdctl`, and Docker/Moby.
 
-## Config versions
-The content of `/etc/containerd/config.toml` must start with a version header, for example:
-```toml
-version = 3
-```
-
-The config version 3 was introduced in containerd v2.0.
-The config version 2 used in containerd 1.x is still supported and automatically
-converted to the config version 3.
```

Comment:
Please keep this section, and summarize refer/link to the new section in the config guide where the config.toml version for cri and containerd is defined